woocommerce/woocommerce-android
# Issue/11323 new product image picker

Closes: #11323 


### Description

Adds a new product image picker within Blaze campaign creation flow. 
- A new item is available in the media picker dialog displayed when editing Ad image
- A new fragment is loaded when selecting the option `Choose existing product photo`

This PR doesn't add any unit tests because there's barely any logic. It is mostly a simple UI and handling navigation. 

### Testing information
1. Trigger Blaze campaign creation flow
2. Select a product that doesn't have any image
3. Tap `Edit Ad` > `Change image` > `Choose existing product photo`
4. Check empty case is handled. 

Repeat the above steps and select a product with multiple images.
1. Select any of the available images and check how the UI is updated accordingly
2. Click save and notice how the Ad image is updated

### Images/gif



https://github.com/user-attachments/assets/5e40d610-d27f-4548-9cdb-92b7340dc01d


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:


323
